src/System.Net.Security/src/System/Net/SecurityStatus.cs

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.Net
+{
+    internal enum SecurityStatus
+    {
+        // Success / Informational
+        OK,
```

Comment:
I always find it worrying when a status enumeration has "success" as the zero value.

"NotSet", or some other otherwise unused value, would be a good thing to put here to ensure that "OK" is always intentional.